### ISSUE TITLE
Community limits should be displayed in the admin panel

### ISSUE THREAD

**Issue by ZJvandeWeg** [flowforge/flowforge#1077]
### Description

When starting an unlicensed FlowForge it has limitations for the community edition. When booting the server a lot message will tell the administrator how far along the limits are:

```
[2022-10-14T10:12:29.895Z] INFO: Usage:
[2022-10-14T10:12:29.895Z] INFO:  Users    : 0/150
[2022-10-14T10:12:29.895Z] INFO:  Teams    : 0/50
[2022-10-14T10:12:29.895Z] INFO:  Projects : 0/50
[2022-10-14T10:12:29.895Z] INFO:  Devices  : 0/50
```

It would be great to also show this on the administrator overview page in the GUI.

**Comment by Jozefik** [comment]
Hey, is this issue free ? I can work on it.

**Comment by knolleary** [comment]
From an API point of view, the existing `/stats` admin endpoint is used by the admin overview page to get this type of information and where the limits info should get added.

**Comment by ZJvandeWeg** [comment]
@Jozefik Jep, totally!

**Comment by Jozefik** [comment]
You want it inside License field or should I create new field ?

**Comment by hardillb** [comment]
I think there are 2 choices: 

1. in the license section
2. in each of the boxes at the top, e.g. users 3/100 meaning 3 of 100 avaiable.


@joepavitt Any comment?

**Comment by Jozefik** [comment]
I have done both. Just tell me which one is better. 

![image](https://user-images.githubusercontent.com/1617822/195853994-acfca077-5b63-4d60-82a5-93766982f989.png)

**Comment by knolleary** [comment]
In the top boxes gets my vote.

Thanks @Jozefik - looks great.

**Comment by Jozefik** [comment]
Okay fine, I will leave it in top boxes. And also before i make PR those limits only apply to Community edition yeah ?

**Comment by knolleary** [comment]
A licensed version of the platform will have its own limits to report - so the `/stats` admin end point should be returning what the limits are (regardless of whether they are community or enterprise licensed).

**Comment by Jozefik** [comment]
Yeah i added limits to `/stats` but comunity license doesn't provide that info with `app.license.get()` it only returns `null`.

**Comment by knolleary** [comment]
I think stats should return for both types. Need to remind myself how the community defaults are picked up. Let me know if you want any pointers.

**Comment by Jozefik** [comment]
@knolleary I will make PR and you can check how i fixed it. Also if I can ask, is it better to fork and make PR or create new branch and make PR ?

**Comment by joepavitt** [comment]
> is it better to fork and make PR or create new branch and make PR ? 

Either is fine, although our general practice is a branch

**Comment by hardillb** [comment]
I think you'll have to fork and PR as you won't have access to push a branch directly to the repo

**Comment by Jozefik** [comment]
Okay, i will fork and make PR

**Comment by knolleary** [comment]
Closed via #1082

**Comment by hardillb** [comment]
Verified on Staging
